packages/netlify-cms-ui-default/src/AuthenticationPage.js

```diff
@@ -95,7 +95,7 @@ AuthenticationPage.propTypes = {
   loginErrorMessage: PropTypes.node,
   renderButtonContent: PropTypes.func,
   renderPageContent: PropTypes.func,
-  t: PropTypes.func.isRequired,
+  t: PropTypes.func,
```

Comment:
`t` is actually required

Reply:
Requiring this throws an error during login with Azure.

Reply:
We should pass the `t` function to the component to fix the error. I think it should be fixed here:
https://github.com/netlify/netlify-cms/pull/4427/files#diff-15ef45b7f567d47fc75af46ae70e53100f7ff3a3fa2b61a07809c8c4fed34e37R63